DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/ManageCourse/EditLearningPathwayDefaults.cshtml

```diff
@@ -0,0 +1,86 @@
+  <div class="nhsuk-grid-column-full">
+    @if (errorHasOccurred) {
+      <vc:error-summary order-of-property-names="@new []{ nameof(EditLearningPathwayDefaultsViewModel.CompleteWithinMonths), nameof(EditLearningPathwayDefaultsViewModel.ValidityMonths) }" />
+    }
+
+    <h1 class="nhsuk-heading-xl">Edit Learning Pathway defaults</h1>
+
+    <form class="nhsuk-u-margin-bottom-3" method="post"
+          asp-controller="ManageCourse"
+          asp-action="SaveLearningPathwayDefaults"
+          asp-route-customisationId="@Model.CustomisationId">
+      <div class="nhsuk-form-group">
+        <vc:text-input asp-for="@nameof(Model.CompleteWithinMonths)"
+                       label="Completion within (in months)"
+                       populate-with-current-value="true"
+                       type="text"
+                       spell-check="false"
+                       autocomplete=""
+                       hint-text="Within how many months of enrolment should the learner complete the course?"
+                       css-class="nhsuk-input nhsuk-input--width-10" />
+      </div>
+
+      <div class="nhsuk-form-group">
+        <vc:text-input asp-for="@nameof(Model.ValidityMonths)"
+                       label="Completion valid for (in months)"
+                       populate-with-current-value="true"
+                       type="text"
+                       spell-check="false"
+                       autocomplete=""
+                       hint-text="For how many months after completion will the completion remain valid
+                                                      (before the learner is expected to complete the course again)?"
+                       css-class="nhsuk-input nhsuk-input--width-10" />
+      </div>
+
+      <div class="nhsuk-checkboxes nhsuk-u-margin-bottom-3">
+
+        <div class="nhsuk-checkboxes__item">
+          <input class="nhsuk-checkboxes__input"
+                 type="checkbox"
+                 id="Mandatory"
+                 asp-for="Mandatory">
+          <label class="nhsuk-label nhsuk-checkboxes__label" for="Mandatory">
+            Course is mandatory
+          </label>
+          <div class="nhsuk-hint nhsuk-checkboxes__hint">
+            Cannot be removed from the current course list by learner after enrolment
+          </div>
+        </div>
+        <div class="nhsuk-checkboxes__item">
+          <input class="nhsuk-checkboxes__input"
+                 type="checkbox"
+                 id="AutoRefresh"
+                 asp-for="AutoRefresh">
+          <label class="nhsuk-label nhsuk-checkboxes__label" for="AutoRefresh">
+            Enable auto-refresh
+          </label>
+          <div class="nhsuk-hint nhsuk-checkboxes__hint">
+            After the completion valid period expires, should the learner automatically be enrolled on a refresher course?
+          </div>
+        </div>
+      </div>
+      <button class="nhsuk-button" type="submit">Save</button>

```

Comment:
Looking at the screenshots, I think this button could do with a little extra space between it and the checkboxes.

Reply:
See updated screenshot in above comment, let me know how that looks :)